ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13.yaml

```diff
@@ -30,21 +25,24 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: smoke-rosa-rhods
+- as: smoke-rosa-rhods-addon-baseline
   cron: 0 12 * * 6
   steps:
     cluster_profile: aws-managed-cspi-qe
     env:
       ADDON1_CONFIG: name=managed-odh;notification-email=interop-qe-ms@redhat.com
-      CLUSTER_NAME: odh-add-413
-      COMPUTE_MACHINE_TYPE: m5.4xlarge
+      CLUSTER_NAME: ods-ad-all-413
```

Comment:
please add `INSTALL_FROM_IIB: "true"` to make sure we install operators using iib when needed.
(see https://github.com/openshift/release/blob/master/ci-operator/step-registry/operator/install/operator-install-ref.yaml#L41)
please add `-IIB` to the filename (variant os it will be `__v4.13-IIB`)